src/braket/tasks/local_quantum_task.py

```diff
@@ -57,8 +67,12 @@ def result(
 
     def async_result(self) -> asyncio.Task:
         """Get the quantum task result asynchronously.
+
+        Raises:
+            NotImplementedError: Asynchronous local simulation unsupported
+
         Returns:
-            Task: Get the quantum task result asynchronously.
+            asyncio.Task: Get the quantum task result asynchronously.
```

Comment:
This return doesn't appear to be implemented yet. Should it be included in the docstring?

Reply:
Yes and no. This is to indicate the implemented function will return this value. However, this would not be the case if this was ever called.